src/models/users.js

```diff
@@ -28,7 +28,12 @@ module.exports = (sequelize, DataTypes) => {
       password: {
         type: DataTypes.STRING,
         allowNull: false,
-      }
+      },
+      roles: {
+        type: DataTypes.ENUM({
+          values: ["user", "admin", "superadmin", "developer"],
+        }),
+      },
     },
```

Comment:
We finally resolved to go with another direction in this matter. Users roles should be of type STRING, with a default value of "user". This was fixed in branch `OT124-97 `so i suggest we discard this changes.